<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Melosimula</class>
 <widget class="QMainWindow" name="Melosimula">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>80</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>739</width>
    <height>70</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Melosimula</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="SnakeValues">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="MelosimulaRF.qrc">
              <normaloff>:/Icons/Icons/SnakeOptions.png</normaloff>:/Icons/Icons/SnakeOptions.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ODEValues">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="MelosimulaRF.qrc">
              <normaloff>:/Icons/Icons/physics.png</normaloff>:/Icons/Icons/physics.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="GraphicValues">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="MelosimulaRF.qrc">
              <normaloff>:/Icons/Icons/graphicOptions.png</normaloff>:/Icons/Icons/graphicOptions.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Heighfield">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="MelosimulaRF.qrc">
              <normaloff>:/Icons/Icons/heighfield.png</normaloff>:/Icons/Icons/heighfield.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Angles</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ruta">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>23</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="LoadAnglesButton">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="MelosimulaRF.qrc">
              <normaloff>:/Icons/Icons/SelectFile.png</normaloff>:/Icons/Icons/SelectFile.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="reverseSpeedButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="MelosimulaRF.qrc">
              <normaloff>:/Icons/Icons/decreaseSpeed.png</normaloff>:/Icons/Icons/decreaseSpeed.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="StepBWButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="MelosimulaRF.qrc">
              <normaloff>:/Icons/Icons/oneFrameBW.png</normaloff>:/Icons/Icons/oneFrameBW.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SimularButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="MelosimulaRF.qrc">
              <normaloff>:/Icons/Icons/Play.png</normaloff>:/Icons/Icons/Play.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pausaButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="MelosimulaRF.qrc">
              <normaloff>:/Icons/Icons/Pause.png</normaloff>:/Icons/Icons/Pause.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <stylestrategy>PreferDefault</stylestrategy>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="MelosimulaRF.qrc">
              <normaloff>:/Icons/Icons/stop.png</normaloff>:/Icons/Icons/stop.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>14</width>
              <height>14</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="refreshButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="MelosimulaRF.qrc">
              <normaloff>:/Icons/Icons/update.png</normaloff>:/Icons/Icons/update.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stepFWButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="MelosimulaRF.qrc">
              <normaloff>:/Icons/Icons/oneFrameFW.png</normaloff>:/Icons/Icons/oneFrameFW.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SpeedButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="MelosimulaRF.qrc">
              <normaloff>:/Icons/Icons/increaseSpeed.png</normaloff>:/Icons/Icons/increaseSpeed.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="multiplier">
            <property name="maximumSize">
             <size>
              <width>100000</width>
              <height>45</height>
             </size>
            </property>
            <property name="text">
             <string>1.0x</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="MelosimulaRF.qrc">
            <normaloff>:/Icons/Icons/Screenshot.png</normaloff>:/Icons/Icons/Screenshot.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNueva_Simulaci_n"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuEdicion">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSimular"/>
    <addaction name="actionDetener"/>
    <addaction name="separator"/>
    <addaction name="actionSnake"/>
    <addaction name="actionODE"/>
    <addaction name="actionGr_ficos"/>
    <addaction name="actionHeighfield"/>
    <addaction name="separator"/>
    <addaction name="actionLogFile"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAyuda"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuPlugins">
    <property name="title">
     <string>Plugins</string>
    </property>
    <addaction name="actionGenerador_de_ngulos"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEdicion"/>
   <addaction name="menuPlugins"/>
   <addaction name="menuAyuda"/>
  </widget>
  <action name="actionSalir">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNueva_Simulaci_n">
   <property name="text">
    <string>New Simulation</string>
   </property>
  </action>
  <action name="actionAyuda">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionSimular">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionDetener">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionSnake">
   <property name="icon">
    <iconset resource="MelosimulaRF.qrc">
     <normaloff>:/Icons/Icons/SnakeOptions.png</normaloff>:/Icons/Icons/SnakeOptions.png</iconset>
   </property>
   <property name="text">
    <string>Snake</string>
   </property>
  </action>
  <action name="actionODE">
   <property name="icon">
    <iconset resource="MelosimulaRF.qrc">
     <normaloff>:/Icons/Icons/physics.png</normaloff>:/Icons/Icons/physics.png</iconset>
   </property>
   <property name="text">
    <string>ODE</string>
   </property>
  </action>
  <action name="actionGr_ficos">
   <property name="icon">
    <iconset resource="MelosimulaRF.qrc">
     <normaloff>:/Icons/Icons/graphicOptions.png</normaloff>:/Icons/Icons/graphicOptions.png</iconset>
   </property>
   <property name="text">
    <string>Graphics</string>
   </property>
  </action>
  <action name="actionHeighfield">
   <property name="icon">
    <iconset resource="MelosimulaRF.qrc">
     <normaloff>:/Icons/Icons/heighfield.png</normaloff>:/Icons/Icons/heighfield.png</iconset>
   </property>
   <property name="text">
    <string>Heighfield</string>
   </property>
  </action>
  <action name="actionLogFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LogFile</string>
   </property>
  </action>
  <action name="actionGenerador_de_ngulos">
   <property name="text">
    <string>Parametric Angle Generator</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MelosimulaRF.qrc"/>
 </resources>
 <connections/>
</ui>
